scss/previews/_examples.scss

```diff
@@ -0,0 +1,41 @@
+// Examples
+// ========
+
+[data-sassdoc-example] {
+  margin: size('gutter') 0;
+
+  [aria-hidden='true'] {
+    display: none;
+  }
+}
+
+[data-sassdoc='example-tabs'] {
+  @include font-family('code');
+  @include font-size('smaller');
+
+  [aria-controls] {
+    display: inline-block;
+    padding: 0 size('shim');
+  }
+
+  [aria-selected='true'] {
+    @include contrasted('text');
+  }
+}
+
+[data-sassdoc='example-tab-solo'] {
+  @include contrasted('text');
+  display: inline-block;
+  padding: 0 size('shim');
+}
+
+[data-sassdoc-example-panel] {
+  border: 1px solid color('border');
+  max-height: size('rhythm') * 12;
+  max-width: 100%;
+  overflow: auto;
+}
+
+[data-sassdoc-example-panel='output'] {
+  padding: size('shim');
```

Comment:
Yeah, this is another example of some meta-documentation confusing what you see. The output always has padding, so that you don't confuse the frame with the design inside it. In this case, we're demonstrating what a code-block looks like — so this code-block is example output, while the other code-block is part of the actual design. This will not be a common use-case.